Test/task1.c

```diff
@@ -0,0 +1,84 @@
+#include <stdio.h>
+#include <stdbool.h>
+
+unsigned int collection[1000];
+
+unsigned int fibonacci(int number);
+bool fibonacciTest();
+
+int main()
+{
+    if(!fibonacciTest())
+    {
+        return 1;
+    }
+    
+    int sum = 0;
+    
+    for(int i = 0; i < 100; i++)
+    {
+        unsigned int number = fibonacci(i);
+        
+        if(number <= 1000000 && number % 2 == 0)
+        {
+            sum += number;
+        }
+
+    }
+
+    printf("the sum of the Fibonacci elements that are less or equal to one million is %d", sum);
+    
+}
+
+
+unsigned int fibonacci(int number){
```

Comment:
```suggestion
}

unsigned int fibonacci(int number) {
```
Две пустые строки подряд не ставят. А вот пробел перед `{` обязателен